frb_codegen/src/library/codegen/ir/mir/ty/dart_opaque.rs

```diff
@@ -11,7 +11,8 @@ impl MirTypeTrait for MirTypeDartOpaque {
         f: &mut F,
         mir_context: &impl MirContext,
     ) {
-        self.get_delegate().visit_types(f, mir_context)
+        MirType::Primitive(MirTypePrimitive::Isize).visit_types(f, mir_context);
+        MirType::Primitive(MirTypePrimitive::Usize).visit_types(f, mir_context);
```

Comment:
nit: what about

1. Add method `MirTypeDartOpaque.get_delegate_rust` which gives usize, `get_delegate_dart` which gives isize
2. At here (visit types), we call `self.get_delegate_rust().visit_types(); self.get_delegate_dart().visit_types();`
3. At places mentioned below, call get_delegate_dart/get_delegate_rust, instead of hardcoded isize/usize